version1/02 - Object types.md

```diff
@@ -500,6 +501,55 @@ The following notation is used in the tables below:
+| M020      | string    | tittel                           |    x     |             |                    |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                     |
+| M025      | string    | offentligTittel                  |          |             |                    |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                     |
+| M021      | string    | beskrivelse                      |          |             |                    |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                     |
+| M024      | string    | forfatter                        |          |             |                    |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                     |
+|           | string    | publiseringsstatus               |    x     |      x      |                    |  |  x   |  x   |   x   |  x   |      |        | Allowed values: NOT_PUBLISHED, PUBLISHED_METADATA, PUBLISHED_METADATA_AND_DOCUMENTS                                 |
+| M300      | string    | dokumentmedium                   |    x     |      x      | dokumentmedium     |  |  x   |  x   |   x   |      |      |        |                                                                                                                     |
+| M101      | date      | registreringsDato                |          |             |                    |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                     |
+| M103      | date      | dokumentetsDato                  |          |             |                    |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                     |
+| M109      | date      | forfallsdato                     |          |             |                    |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                     |
+| M104      | timestamp | mottattDato                      |          |             |                    |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                     |
```

Comment:
Should we add the three fields to Basis- and Moeteregistrering?

Reply:
I was thinking that we should not advertise them as available for now because there is no use case for that. It's just an implementation detail.